eli5/catboost.py

```diff
@@ -0,0 +1,84 @@
+
+from __future__ import absolute_import,division
+
+import numpy as np
```

Comment:
I think the tests are failing due to missing type annotations for numpy and catboost: https://travis-ci.org/TeamHG-Memex/eli5/jobs/512862671#L777 - the way to fix this is to ignore the types here in this way: ``import numpy as np  # type: ignore``, and same for catboost import. This is something to be improved in our development process, but at the moment it is required, sorry.